### PR TITLE
Commands

### PR DESCRIPTION
**Pull request by harshini188**

### CODE REVIEW

File: gitcom.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+git init: creates new local repo
+git clone: creates local copy of remote repo
+git merge: merges remote branch into current local branch
```
Comment (avinash367) [inline gitcom.txt:3]: i did not understand properly
git merge will merge to branches in remote repo .....is that what you think ?

File: gitcom.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+git branch: creates new branch
+git add: stages new and changed files
```
Comment (avinash367) [inline gitcom.txt:5]: git add . // to add all files

**Review by avinash367 — COMMENTED** [review]
Please look

File: gitcom.txt (hunk 1 above)
Comment (kulkarni90) [inline gitcom.txt:3]: check this command

Comment (kulkarni90) [inline gitcom.txt:3]: Please check this command again

File: gitcom.txt (hunk 2 above)
Comment (kulkarni90) [inline gitcom.txt:5]: I didn't get this command please clarify

**Review by kulkarni90 — COMMENTED** [review]
Please frame the sentences in proper understanding and meaning full way.

File: gitcom.txt (hunk 1 above)
Comment (harshini188) [reply gitcom.txt:3]: ok we will correct it

File: gitcom.txt (hunk 2 above)
Comment (harshini188) [reply gitcom.txt:5]: both are same but in different way

Comment (harshini188) [reply gitcom.txt:5]: git add it says that it stages the file and changed files but not the deleted files

Comment (avinash367) [reply gitcom.txt:5]: when we use git add....the command works properly

**Pull request closed without merging**